### ISSUE TITLE
Question about Train

### ISSUE THREAD

**Issue by begins233** [gafniguy/4D-Facial-Avatars#10]
When I tried to train, a lot of errors appeared. My training steps are as follows:
1. Download the data you provided
2. Modify the data path in the configuration file
3. Run train_transformed_rays.py

When I run the third step, [run_one_iter_of_nerf ](https://github.com/gafniguy/4D-Facial-Avatars/blob/main/nerface_code/nerf-pytorch/nerf/train_utils.py)produced a series of errors because it did not pass in the ray_directions_ablation parameter. At present, I commented all the relevant codes and can start training, but I don't know if it is correct.My training output is below:
```[TRAIN] Iter: 0 Loss: 0.2382410168647766 BG Loss: 0.0 PSNR: 6.2298346592408995 LatentReg: 0.0
[VAL] =======> Iter: 0
Validation loss: 0.10423796623945236 Validation PSNR: 9.81974070622407 Time: 2.703904151916504
================== Saved Checkpoint =================
[TRAIN] Iter: 100 Loss: 0.10412006825208664 BG Loss: 0.0 PSNR: 9.824655558769116 LatentReg: 0.0
[TRAIN] Iter: 200 Loss: 0.03349904716014862 BG Loss: 0.0 PSNR: 14.749675457683708 LatentReg: 0.0
[TRAIN] Iter: 300 Loss: 0.017453620210289955 BG Loss: 0.0 PSNR: 17.581144784878884 LatentReg: 0.0
[TRAIN] Iter: 400 Loss: 0.01741687022149563 BG Loss: 0.0 PSNR: 17.590298842108833 LatentReg: 0.0
[TRAIN] Iter: 500 Loss: 0.029193900525569916 BG Loss: 0.0 PSNR: 15.347078761129636 LatentReg: 0.0

**Comment by gafniguy** [comment]
Yes, sorry about that! That indeed was a last-minute thing I forgot to comment out. Your numbers seem to be in order. Did it work out for you?
